src/model.py

```diff
@@ -0,0 +1,157 @@
+# author: Siqi Tao
+# date: 2021-11-26
+
+"""Perform machine learning analysis on the cleaned data
+Usage: model.py --input=<input>  --output=<output> 
+ 
+Options:
+--input=<input>       The path or filename pointing to the data
+--output=<output>     The prefix where to write the output figure(s)/table(s) to and what to call it 
+"""
+
+import os
+import pandas as pd
+from docopt import docopt
+
+import numpy as np
+import pandas as pd
+from sklearn.compose import ColumnTransformer, make_column_transformer
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import make_scorer
+from sklearn.model_selection import (
+    RandomizedSearchCV,
+    cross_val_score,
+    cross_validate
+)
+from sklearn.pipeline import Pipeline, make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from scipy.stats import loguniform
+from sklearn.metrics import confusion_matrix
+import pickle
+
+opt = docopt(__doc__)
+
+def main(): 
+    
+    # Get data
+    input = opt["--input"]
+    train_df = pd.read_csv(f"{input}train.csv", low_memory=False)
```

Comment:
Change this line to train_df = pd.read_csv(f"{input}train.csv", low_memory=False).set_index("index").rename_axis(None)

Reply:
done